request/src/main/java/com/vimeo/networking2/VimeoService.kt

```diff
@@ -21,163 +21,167 @@
  */
 package com.vimeo.networking2
 
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_ACTIVE
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_APP_TYPE
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_AUTH_CODE
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_CLIENT_ID
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_COMMENT_TEXT_BODY
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_GET_FIELD_FILTER
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_PASSWORD
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_USERS_BIO
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_USERS_LOCATION
+import com.vimeo.networking2.ApiConstants.Parameters.PARAMETER_USERS_NAME
+import com.vimeo.networking2.enums.ConnectedAppType
+import com.vimeo.networking2.enums.NotificationType
 import com.vimeo.networking2.internal.VimeoCall
 import com.vimeo.networking2.params.BatchPublishToSocialMedia
 import com.vimeo.networking2.params.ModifyVideoInAlbumsSpecs
 import com.vimeo.networking2.params.ModifyVideosInAlbumSpecs
 import okhttp3.CacheControl
 import retrofit2.http.Body
 import retrofit2.http.DELETE
+import retrofit2.http.Field
 import retrofit2.http.GET
 import retrofit2.http.Header
 import retrofit2.http.Headers
 import retrofit2.http.PATCH
 import retrofit2.http.POST
 import retrofit2.http.PUT
 import retrofit2.http.Path
+import retrofit2.http.Query
 import retrofit2.http.QueryMap
 import retrofit2.http.Url
 
 /**
  * Retrofit service for the Vimeo API.
  *
- * TODO: Add documentation to functions.
+ * @see VimeoApiClient
  */
 @Suppress("unused", "UndocumentedPublicFunction", "ComplexInterface")
```

Comment:
Can we remove "unused" from the Supression? Assuming since this class is internal all its method should be exposed via `VimeoApiClient` right?

Removing it for me locally shows `getVideoStatus()`  is the only unused method, this should be expose via the client right?